lang/java/integration-test/codegen-test/src/test/java/org/apache/avro/codegentest/TestCustomConversion.java

```diff
@@ -18,26 +18,44 @@
 
 package org.apache.avro.codegentest;
 
+import org.apache.avro.LogicalTypes;
 import org.apache.avro.codegentest.testdata.LogicalTypesWithCustomConversion;
+import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.IOException;
 import java.math.BigInteger;
 
 public class TestCustomConversion extends AbstractSpecificRecordTest {
 
+  @BeforeClass
+  public static void init() {
+    LogicalTypes.register(FixedSizeStringFactory.NAME, new FixedSizeStringFactory());
+  }
+
   @Test
-  public void testNullValues() throws IOException {
+  public void testNullValues() {
     LogicalTypesWithCustomConversion instanceOfGeneratedClass = LogicalTypesWithCustomConversion.newBuilder()
-        .setNonNullCustomField(new CustomDecimal(BigInteger.valueOf(100), 2)).build();
+        .setNonNullCustomField(new CustomDecimal(BigInteger.valueOf(100), 2))
+        .setNonNullFixedSizeString(new FixedSizeString("test")).build();
     verifySerDeAndStandardMethods(instanceOfGeneratedClass);
   }
 
   @Test
-  public void testNonNullValues() throws IOException {
+  public void testNonNullValues() {
     LogicalTypesWithCustomConversion instanceOfGeneratedClass = LogicalTypesWithCustomConversion.newBuilder()
         .setNonNullCustomField(new CustomDecimal(BigInteger.valueOf(100), 2))
-        .setNullableCustomField(new CustomDecimal(BigInteger.valueOf(3000), 2)).build();
+        .setNullableCustomField(new CustomDecimal(BigInteger.valueOf(3000), 2))
+        .setNonNullFixedSizeString(new FixedSizeString("test")).setNullableFixedSizeString(new FixedSizeString("test2"))
```

Comment:
Move `.setNullableFixedSizeString(new FixedSizeString("test2"))` to new line for readability

Reply:
Good observation.
Unfortunately these changes broke the build due to `spotless-maven-plugin` format violation. The issue only happened during the `avro-parent` project build.